### DISCUSSION TITLE
Before&After: Enschede city & Sloped roofs

### DISCUSSION THREAD

**Discussion by ngsrinidhi** [stefanfluck/palmpy#79] [Show and tell]
<img width="808" alt="Screenshot 2023-09-11 at 09 17 20" src="https://github.com/stefanfluck/palmpy/assets/10849110/8ceb7531-adc9-495f-8a74-d2d5218793a5">

@stefanfluck thanks for a very nice and useful package! 
I wanted to share the 'before & after' image of the PALM run I made for the Dutch city of Enschede. 

I also wanted to take this opportunity to ask your thoughts on the way to include 'sloped' roofs in a 'stepped way' that PALM uses. 
I see in the code that we can provide it as the 'buildings_3d' variable, but if you have thought about the way to do this, let me know.

**Comment by stefanfluck** [comment]
Hey!! Thank you so much for that image, this is awesome!

About the dobuildings3d namelist parameter, yes, it would make sense to extend the code in a way that a raster can be given with the top building height. Currently the dobuildings3d parameter creates the buildings_3d variable and respects the HEIGHT_BOT attribute in the shapefile, so overhanging polygons can be made. 

What you could try is to split your polygon buildings into several with the same building id, but with different height. Should work if I recall correctly.
